lib/dom.dart

```diff
@@ -1058,14 +1062,14 @@ class FilteredElementList extends IterableBase<Element>
   // bug where ListMixin<E>.indexOf() expects Object as the argument.
   @override
   int indexOf(Object element, [int start = 0]) =>
-      _filtered.indexOf(element, start);
+      _filtered.indexOf(element as Element, start);
```

Comment:
Is the above TODO still an issue or can we type the arg as Element? (ditto for next one)

Reply:
Still an issue and likely wont' be changed for a while. Would you prefer I remove the comments or update them to point to https://github.com/dart-lang/sdk/issues/31311

Reply:
Seems like just remove the TODO as it is unlikely to get resolved based on that issue

Reply:
I added a comment so that it's clear why there is a cast, but it's not marked as a TODO anymore.